### PR TITLE
Story/flcrm 11343 add system status column to record

### PR DESCRIPTION
**Pull request by HenryTabima**
add system status column to record

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fulcrumapp/fulcrum-schema",
-  "version": "3.6.2",
+  "version": "3.7.2",
```
Comment (osbornk) [inline package.json]: Why 3.7.2? It should just be 3.7.0 since it is a new non-breaking feature.

**Comment by osbornk** [comment]
This looks good so far. I would test this out by running schema-service with this commit. Then test update Fulcrum so that the default version is v5. Create an app. Does it work?

File: package.json (hunk 1 above)
Comment (HenryTabima) [reply package.json]: yes you are right

File: src/schemas/v5.js [hunk 2]
```diff
@@ -0,0 +1,2378 @@
...
+  {
+    name: "text_value",
+    type: "string",
+    length: 500,
+  },
+  {
+    name: "number_value",
+    type: "double",
+  },
+  {
```
Comment (osbornk) [inline src/schemas/v5.js]: Move to `systemFormTableColumns`.

**Review by m1g — APPROVED** [review]
LGTM 🚢

**Review by osbornk — APPROVED** [review]

**Pull request merged**